pycqed/instrument_drivers/meta_instrument/heterodyne.py

```diff
@@ -160,40 +170,23 @@ def prepare_CBox(self, get_t_base=True):
         # self.CBox.set('acquisition_mode', 'idle') # aded with xiang
 
     def prepare_UHFQC(self):
-        if self.AWG != None:
-            if (self._awg_seq_filename not in self.AWG.setup_filename() or
-                    self._awg_seq_parameters_changed) and \
-                    self.auto_seq_loading():
-                self._awg_seq_filename = \
-                    st_seqs.generate_and_upload_marker_sequence(
-                        5e-9, self.trigger_separation(), RF_mod=False,
-                        acq_marker_channels=self.acq_marker_channels())
-                self._awg_seq_parameters_changed = False
-
-        if self._UHFQC_awg_parameters_changed and self.auto_seq_loading():
-            self._acquisition_instr.awg_sequence_acquisition()
-            self._UHFQC_awg_parameters_changed = False
-
-        # prepare weights and rotation
```

Comment:
I recall a discussion with @NielsBultink that led me to settle on this version of the prepare statement. I recall it has something to do with #135  and #94 . However I cannot recall the exact argument and I would argue you are right.